index.html

```diff
@@ -205,20 +261,26 @@ <h4>Start and stop kubernetes</h4>
       <div class="row">
         <div class="col-8">
           <hr>
-          <h4>Turn on standard services</h4>
+          <h4>
+            Turn on standard services
+          </h4>
           <div class="p-code-copyable">
             <input class="p-code-copyable__input" aria-label="microk8s.enable dashboard registry istio [...]" value="microk8s.enable dashboard registry istio [...]" readonly="readonly">
           </div>
-          <p>Enable and disable particular standard add-on services using the <code>microk8s.enable</code> and <code>microk8s.disable</code> commands. See the <code>--help</code> for details.</p>
+          <p>
+            Enable and disable particular standard add-on services using the <code>microk8s.enable</code> and <code>microk8s.disable</code> commands. See the <code>microk8s.enable --help</code> for details.
+          </p>
         </div>
       </div>
       <div class="row">
         <div class="col-8">
           <hr>
-          <h4>Try a different version, or a beta, or a daily build</h4>
-          <p>Snaps are published in <a href="https://docs.snapcraft.io/channels/551">channels</a> which are made up
-            of a track (or major version), and an expected level of stability. Try <code>snap info microk8s</code> to
-            see what versions are currently published. For example:</p>
+          <h4>
+            Try a different version, or a beta or a daily build
+          </h4>
+          <p>
+            Snaps are published in <a href="https://docs.snapcraft.io/channels/551">channels</a> which are made up of a track (or major version), and an expected level of stability. Try <code>snap info microk8s</code> to see what versions are currently published. For example:
```

Comment:
https://snapcraft.io/docs/channels would be a nicer representation of the docs post.